tox.ini

```diff
@@ -1,20 +1,20 @@
 [tox]
 envlist =
-    py3{4,5,6}
+    py3{5,6}
     pypy3
 skip_missing_interpreters = True
 
 [testenv]
 deps =
-    cassandra-driver
     flake8
     isort
 commands =
-    flake8 --show-source aiocassandra.py setup.py tests.py
-    isort --check-only aiocassandra.py setup.py tests.py
+    flake8 --show-source aiocassandra.py setup.py
+    isort --check-only aiocassandra.py setup.py
+    flake8 --show-source tests
+    isort --check-only -rc tests --diff
 
-    # envpython to mute "WARNING:test command found but not installed in testenv"
-    {envpython} tests.py
+    {envpython} setup.py pytest
```

Comment:
In fact, using setup.py didn't make it to become a standard protocol for running tests. Turns out that the only best practice nowadays is to just invoke pytest.